### DISCUSSION TITLE
Natures Shaders for Unity 6.0?

### DISCUSSION THREAD

**Discussion by krazyxiii** [visualdesigncafe/nature-renderer#84] [General]
Hi,

I see that there is a Nature Renderer released for Unity 6, will there be a release for Unity 6 for Nature Shaders or is Nature Shaders 2022 compatible with Unity 6? Please reply as this is a critical factor in the planning of my project. Thank you.

**Comment by maxplooi** [comment]
Unity made a lot of significant changes to the render pipelines which caused the Nature Shaders to break. I felt that it was not helpful to constantly delay Nature Renderer updates because the Nature Shaders are not working, even though Nature Renderer itself works perfectly. It was hurting all the developers who are using Nature Renderer with their own shaders. So, I decided to no longer bundle the shaders with Nature Renderer and keep them as a separate asset.

Once I figure out how to get the shaders compatible with Unity 6 there will be a new release of the shaders. No idea how long it will take though, updating these shaders requires completely reverse-engineering the render pipelines in Unity since Unity does not provide any documentation on their shaders.

**Reply by xxsmbr** [reply]
any update on this?
